server/webapp/WEB-INF/rails/webpack/views/pages/new_plugins/spec/plugins_widget_spec.tsx

```diff
@@ -15,16 +15,17 @@
  */
 
 import * as m from "mithril";
+import {PluginInfos} from "models/shared/plugin_infos_new/plugin_infos";
 import {PluginsWidget} from "../plugins_widget";
 
 import * as collapsiblePanelStyles from "../../../components/collapsible_panel/index.scss";
 import * as keyValuePairStyles from "../../../components/key_value_pair/index.scss";
 
 describe("New Plugins Widget", () => {
   const simulateEvent = require("simulate-event");
-  const PluginInfos   = require("models/shared/plugin_infos");
 
-  const pluginInfos = PluginInfos.fromJSON([getEAPluginInfo(), getNotificationPluginInfo()]);
+  const data        = [getEAPluginInfo(), getNotificationPluginInfo()];
+  const pluginInfos = PluginInfos.fromJSON({_embedded: {plugin_info: data}});
```

Comment:
@GaneshSPatil @ketan Somehow, the `fromJSON` method in the older PluginInfos was able to accept data without an `_embedded` object in the json. Do we need to support the same behaviour in the new code? I think not, so I have just created this object with an `_embedded`, as it should be received from the API.